scripts/update-1.8.2-1.8.3.sql

```diff
@@ -0,0 +1,31 @@
+-- This script contains multiple comment blocks that can update
```

Comment:
The version numbers in this file seem inconsistent.

1.8.2 and 1.8.3, 1.6.2 and 1.7.0, 1.9.1